locale/fi/text.json

```diff
@@ -0,0 +1,121 @@
+{
+  "title": "Teksti (Text)",
+  "intro": "",
+  "note.title": "",
+  "note.items": ["", "", ""],
+  "sections": [
+    {
+      "title": "Kirjasintyyppi (Font)",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Suomi.fi-tyyleissä sekä käyttöliittymäkomponenteissa käytetty kirjasin Source Sans Pro on Googlen välittämä open source fontti, kaikki sen leikkaukset ovat ilmaisia ja vapaasti käytettävissäsi (Open Font License). Kirjasimen voi ladata omalle työasemalle Google Fontsista."
+        },
+        {
+          "image.key": "",
```

Comment:
I really dislike the habit of leaving empty structures in translation files. In my opinion it's a lot of hassle trying to figure out what goes where when there are a lot of empty placeholders. And not such a big deal to add new structures when content changes.

The only thing that favours this kind of style is the possibility for non technical people to add text, but in any case, a release is required which involves the development workflow. 

I think we should think about this in a bit more detail and try to come up with a better workflow.

Reply:
True, I have just continued the convention that was to be used when creating translation files.

I would also want to see much more simpler structure in translation files. Also to use unique key-names for translation files if ever possible, much more easier to find the correct  place.

Reply:
I'm down with coming up with a better solution for this. It's not a complete mess as it is, but could be cleaner.

Reply:
Let's create a story for investigating this and add it to the improvements H1 epic?